packages/deepbook/sources/book/fill.move

```diff
@@ -96,7 +96,11 @@ module deepbook::fill {
     }
 
     public(package) fun quote_quantity(self: &Fill): u64 {
-        self.quote_quantity
+        if (self.expired) {
+            0
```

Comment:
^

Reply:
this should return `self.quote_quantity`, it should not return 0 if expired. 
https://github.com/MystenLabs/deepbookv3/pull/186/files#diff-a943a789a174807304fcffb16c7def5021bed18a28d56ce341f9656b82923c4bL98

Reply:
Ok post-audit didn't get that change, I'll update